app/CMakeLists.txt

```
@@ -173,19 +173,22 @@ endif()
 # DEMO: geometry tracking
 #-----------------------------------------------------------------------------#
 
-if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_CUDA AND CELERITAS_USE_VecGeom)
+if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_CUDA)
   add_executable(demo-rasterizer
     demo-rasterizer/demo-rasterizer.cc
     demo-rasterizer/RDemoRunner.cc
     demo-rasterizer/RDemoKernel.cu
     demo-rasterizer/ImageIO.cc
     demo-rasterizer/ImageStore.cc
   )
-  celeritas_target_link_libraries(demo-rasterizer
+  set(_rasterizer_libs
     Celeritas::celeritas
     nlohmann_json::nlohmann_json
-    VecGeom::vecgeom
   )
+  if(CELERITAS_USE_CUDA AND CELERITAS_DEBUG AND CELERITAS_GEO STREQUAL "VecGeom")
```

```suggestion
  if(CELERITAS_GEO STREQUAL "VecGeom")
```
as the statement is already is a `CELERITAS_USE_CUDA is true` section and I don't see why you need the `VecGeom` libraries only in debug mode.